docs/changelog.rst

```diff
@@ -152,23 +152,23 @@ Bugfixes - 4.0.0a7
   without the ``testenv:`` prefix - by :user:`jugmac00`, :user:`obestwalter`, :user:`eumiro`. (`#1985 <https://github.com/tox-dev/tox/issues/1985>`_)
 - Fix legacy list env command for empty/missing envlist - by :user:`jugmac00`. (`#1987 <https://github.com/tox-dev/tox/issues/1987>`_)
 - Requirements and constraints files handling got reimplemented, which should fix all open issues related to this area
-  - by :user:`gaborbernat`. (`#1991 <https://github.com/tox-dev/tox/issues/1991>`_)
+  - by :user:`gaborbernat`. (`bugfix #1991 <https://github.com/tox-dev/tox/pull/1991>`_)
```

Comment:
I do not see a pattern when the classifiers are added and when not.

Reply:
See above.